pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,102 @@
+---
+$title: Use AMP components in non-AMP pages
+$order: 1
+description: 'Use a subset of AMP components as standalone elements in non-AMP pages with Bento AMP.'
+tutorial: false
+toc: true
+formats:
+  - websites
+author: CrystalOnScript
+---
+
+Use a subset of AMP components as standalone elements in non-AMP pages with Bento AMP. Bento AMP is a project that embraces use of AMP components without the need to commit to fully valid AMP. This creates copy and past-able code for common UI widgets, allows testing if AMP is the right switch for your website, and using Bento in a migration plan. 
+
+[tip type="important"]
+ Bento is only available [experimentally](../learn/experimental.md). Read our [Blog Post](TODO) for information on future plans and how to get involved! 
+[/tip]
+
+# Use Bento components in non-AMP pages
+
+Bento components are available for experimental use in the developer preview.
+
+
+## Enable Bento Experiment 
+
+Bento components are available experimentally and you must [enable experimental features](../learn/experimental.md).
+
+You can enable all Bento components by including the script below:
+
+
+```
+<script>
+  (self.AMP = self.AMP || []).push(function (AMP) {
+    AMP.toggleExperiment('bento', true);
+  });
+</script>
+```
+
+
+Or enable a single Bento component by including the script below. Ensure you replace `bento-foo` with the desired [supported component](#available-bento-components) name given on its documentation page:
+
+
+```
+<script>
+  (self.AMP = self.AMP || []).push(function (AMP) {
+    AMP.toggleExperiment('bento-foo', true);
+  });
+</script>
+```
+
+
+
+## Import AMP runtime and Bento component logic
+
+You must include the AMP runtime script and import the logic for each individual Bento component desired.
+
+```
+<script async src="https://cdn.ampproject.org/v0.js"></script>
+
+<script async custom-element="amp-bento" src="https://cdn.ampproject.org/v0/amp-bento-1.0.js"></script>
+``` 
+
+Read each Bento component’s reference documentation for implementation details.
+
+
+# Available Bento components 
+
+Bento supported AMP components are listed below:
+
+
+
+*   [amp-accordion](../../../documentation/components/reference/amp-accordion.md)
+*   [amp-base-carousel](../../../documentation/components/reference/amp-base-carousel.md)
+*   [amp-inline-gallery](../../../documentation/components/reference/amp-inline-gallery.md)
+    *   amp-inline-gallery-pagination
+    *   amp-inline-gallery-thumbnails
+*   [amp-stream-gallery](TODO)
```

Comment:
Note that this is in AMPHTML: https://github.com/ampproject/amphtml/blob/master/extensions/amp-stream-gallery/amp-stream-gallery.md

Reply:
got it, thank you!